man/pdfpcrc.in

```diff
@@ -65,11 +65,11 @@ Percentage of the presenter screen to be used for the current slide. (int, Defau
 .TP
 .BI disable-caching
 see
-.B pdfpcrc(5)
+.B pdfpc(1)
```

Comment:
The information about `disable-caching` is in pdfpcrc(5)

Reply:
Sorry, I am probably confused. Where is it in pdfpcrc(5)? I rather see it described in pdfpc(1), section "Caching / Pre-rendering" under "FEATURES". In any case, self-referring pdfpcrc(5) from within pdfpcrc(5) is strange.

Reply:
Oh sorry! You are right and I am were confused. I was thinking you changed this in pdfpc(1)... So, yes the reference to pdfpc(1) is correct here!